src/liger_kernel/transformers/monkey_patch.py

```diff
@@ -1155,7 +1157,7 @@ def apply_liger_kernel_to_paligemma(
 
         language_model = model.language_model
 
-        if isinstance(language_model, GemmaForCausalLM):
+        if isinstance(language_model, GemmaForCausalLM) or isinstance(language_model, GemmaModel):
```

Comment:
nit: [`isinstace(object, classinfo)`](https://docs.python.org/3/library/functions.html#isinstance) accepts *classinfo* as a tuple
```suggestion
        if isinstance(language_model, (GemmaForCausalLM, GemmaModel)):
```